frontend/src/domain/routes/Routes.tsx

```diff
@@ -58,7 +60,15 @@ const Routes = () => {
                 </Popup>
               </Marker>
             ))}
-            <Polyline positions={drawLine(route?.stops)} />
+            {route?.shapes.map((shape: Shape) => (
```

Comment:
This is only a frontend for example/demo purposes, but would this change also mean that a route without a shape will not be rendered? Should we gave the `Polyline` as a backup? :shrug:

Reply:
Yeah good idea, let's do that 👍